numba/core/inline_closurecall.py

```diff
@@ -1382,12 +1452,15 @@ def inline_array(array_var, expr, stmts, list_vars, dels):
         # Create a variable to hold the numpy empty function.
         empty_func = ir.Var(scope, mk_unique_var("empty_func"), loc)
         fnty = get_np_ufunc_typ(np.empty)
-        sig = context.resolve_function_type(fnty, (size_typ,), {'dtype':nptype})
+        # XXX: Does this do anything?
```

Comment:
Could well be a side effect that's needed. If you remove it, do all the tests still pass?